lib/state.js

```diff
@@ -460,7 +460,8 @@ StateResolver.prototype.getMirrorProperties_ = function(mirror) {
 
 StateResolver.prototype.resolveMirrorProperty_ = function(property) {
   var name = String(property.name());
-  if (property.isNative()) {
+  var isArrayLen = property.mirror_.isArray() && name === 'length';
```

Comment:
It should be some subtype of `PropertyMirror` which defines the `mirror_` field: https://github.com/v8/v8/blob/master/src/debug/mirrors.js#L1466.